logging_output_scripts/latex_tabulars.py

```diff
@@ -0,0 +1,148 @@
+import json
+from tabulate import tabulate
+import pandas as pd
+
+"""
+This script uses tabulate package to create LaTex tables based on the values
+calculated in Summary_csv.py (Except for Genomes-Tables which use a Json)
+"""
+
+dirs = {0: 'OBR', 1: 'UBR', 2: 'CSR', 3: 'MPR'}
+
+# *_short Dictionaries are used in the tables headers whereas other dicts are used to access data from file
+datasets = {0: 'parkinson_total', 1: 'protein_structure', 2: 'airfoil_self_noise',
+            3: 'concrete_strength', 4: 'combined_cycle_power_plant'}
+
+# Empty string needed for formatting purposes
+datasets_short = {-1: ' ', 0: 'PT', 1: 'PTTS', 2: 'ASN',
+                  3: 'CS', 4: 'CCPP'}
+
+comp_column = {0: 'MEAN_COMP', 1: 'STD_COMP', 2: 'MEDIAN_COMP', 3: 'MIN_COMP', 4: 'MAX_COMP'}
+comp_column_short = {0: 'mean', 1: 'standard deviation', 2: 'median', 3: 'min', 4: 'max'}
+
+thresh_column = {0: 'THRESH_0_MEAN', 1: 'THRESH_0_MIN', 2: 'THRESH_0_MAX', 3: 'THRESH_1_MEAN', 4: 'THRESH_1_MIN',
+                 5: 'THRESH_1_MAX', 6: 'THRESH_2_MEAN', 7: 'THRESH_2_MIN', 8: 'THRESH_2_MAX'}
+thresh_column_short = {0: 'mean', 1: 'min', 2: 'max', 3: 'mean', 4: 'min',
+                       5: 'max', 6: 'mean', 7: 'min', 8: 'max'}
+
+iter_column = {0: 'FIN_ITER_MAX', 1: 'FIN_ITER_MIN', 2: 'FIN_ITER_MEAN'}
+iter_column_short = {0: 'max', 1: 'min', 2: 'mean'}
+
+
+def load_problem_columns(df, c):
+    data_res = []
+    for v in datasets.values():
+        res = df[df['Problem'].str.contains(v)]
+        data_res.append(float(res[c]))
+        return data_res
+
+
+# COMPLEXITY TABLES
+def write_complexity():
+    for i in dirs:
+        directory = dirs[i]
+        df = pd.read_csv(f"../{directory}/Results.csv")
+        comp_list = []
+        for j in comp_column:
+            c = comp_column[j]
```

Comment:
Why not use `for c in comp_column.values()`? Then you can remove line 47

Reply:
The index is needed to access the other directory containing name for the actual latex tabular. Switched to iterating over both dictionaries simultaneously using zip